server/controllers/finance/patientInvoice.js

```diff
@@ -100,7 +100,8 @@ function lookupInvoice(invoiceUuid) {
       project.abbr, invoice.reference) AS reference, invoice.cost,
       invoice.description, BUID(invoice.debtor_uuid) AS debtor_uuid,
       patient.display_name AS debtor_name,   BUID(patient.uuid) as patient_uuid,
-      invoice.user_id, invoice.date, user.display_name, invoice.service_id,
+      invoice.user_id, invoice.date, invoice.created_at, user.display_name, invoice.service_id,
+      DATEDIFF(invoice.date, invoice.created_at) as date_difference,
```

Comment:
To save modifying the JS, you could just do something like;
```
IF(DATEDIFF(invoice.date, invoice.created_at), TRUE, FALSE) AS has_delay
```
or something similar.  Then you would just need to change the handlebars templates to do `{{if has_delay}}`.